app/api/report.py

```diff
@@ -204,8 +204,29 @@ def get_manager_weekly_list(weekly_id=None):
         }, upsert=False)
 
         return jsonify(ret)
+    
+    
 
-
+@bp.route("/overall_reviewes/<string:user_id>", methods=["GET"])
+@jwt_required
+def overall_reviewes(user_id):
+    docs = mongo.db.reports.find({"user":user_id})
+    user = mongo.db.users.find_one({"_id":ObjectId(user_id)})
+    weights = user['managers']
+    manager_weight = weights[0]

```

Comment:
why 0 index?

Reply:
because the manager has only a single weight. so we need to find only weight value which is in this format [{id:..., weight:1}]  so in this list, only one dictionary so applied here index 0. in every condition, the manager has only a single weight.